package/windows/utils.psm1

```diff
@@ -156,6 +156,25 @@ function Invoke-HNSRequest
     return $output;
 }
 
+function Transfer-File
+{
+    param (
+        [parameter(Mandatory = $true)] [string]$Src,
+        [parameter(Mandatory = $true)] [string]$Dst
+    )
+
+    if (Test-Path -PathType leaf -Path $Dst) {
+        $dstHasher = Get-FileHash -Path $Dst
+        $srcHasher = Get-FileHash -Path $Src
+        if ($dstHasher.Hash -eq $srcHasher.Hash) {
+            return $false
+        }
+    }
+
+    $null = Copy-Item -Force -Path $Src -Destination $Dst
+    return $true
```

Comment:
don't know much about windows; do we return the actual result of the command execution? Would false be returned, and error logged, if the command had failed?

Reply:
Calling `Copy-Item` without `-ErrorAction Ignore`, if it has failed(for example, the file handler of the destination file is using by another process), it will throw an error.

I will change here to wrap by a `try...catch...` block, then output some useful message.